### DISCUSSION TITLE
What do I need to do if I want to carve a rigid body instead of a flexible one?

### DISCUSSION THREAD

**Discussion by MingSongJiang** [InfinyTech3D/SofaUnity#117] [Help - Using SofaUnity] [answered]
Hello,

What do I need to do if I want to carve a rigid body instead of a flexible one?

Thanks

**Comment by epernod** [comment, marked as answer]
Hi,

you just need to define your 3D object as rigid in your SOFA scene, without physical properties like FEM and numerical integration.
You should go through the SOFA documentation or tutorials to understand how to manage that.

Best regards
